op-node/rollup/derive/l1_block_info.go

```diff
@@ -18,8 +18,9 @@ import (
 )
 
 const (
-	L1InfoFuncBedrockSignature = "setL1BlockValues(uint64,uint64,uint256,bytes32,uint64,bytes32,uint256,uint256)"
+	L1InfoFuncBedrockSignature = "setL1BlockValues(uint64,uint64,uint256,bytes32,uint64,bytes32,uint256,uint256,uint8,uint256[])"
```

Comment:
No diff to this function sig, it will break things

Reply:
but we are updating `setL1BlockValues` in L1Block.sol with a new argument, `_dependencySet`. don't we need to update `L1InfoFuncBedrockSignature` here?

Reply:
We should not be updating `setL1BlockValues` at all - this would break legacy chains. The idea is that we only expand the abi of L1Block contract